app/services/export_service_exporters/response_exporter.rb

```diff
@@ -36,6 +37,14 @@ def export_file(response:, to:, ext:, type:)
       stored_file.download_blob_to File.join(to, fn)
     end
 
+    def export_file_as_attachment(response:, to:, ext:, type:, optional: false)
```

Comment:
5 arguments for a method? I think the recommended amount is 3 max ;)

Reply:
its a private method - it just saves a lot of duplication

Reply:
also, they are keyword arguments which would normally be a hash of options so only really count as 1.  The max number of arguments recommended would be for positional arguments as it is very hard to read when you have loads.  Ive just been in an aws-sdk-s3 method that has 18 keyword arguments to put it in perspective !